packages/client/src/parseAbiValues.ts

```diff
@@ -0,0 +1,21 @@
+import { toHexString } from "./toHexString";
+
+export const parseAbiValues = (types: string[], values: string[]) => {
```

Comment:
We could add unit tests for these types of standalone functions in the near future.

Reply:
https://github.com/zeta-chain/toolkit/issues/232